db/migrate_cards/20180718180138_new_homepage_codenames.rb

```diff
@@ -0,0 +1,50 @@
+# -*- encoding : utf-8 -*-
+
+class NewHomepageCodenames < Card::Migration
+  def up
+    ["homepage numbers", "homepage projects",
+     "homepage topics", "homepage organizations",
+     "homepage video section", "homepage video container",
+     "homepage solution text", "homepage project text",
+     "homepage topic text", "homepage footer",
+     "menu: explore", "menu: about", "menu: connect", "menu: legal",
+     "newsletter signup"].each do |name|
+      ensure_card name, codename: codename(name)
+    end
+    # ["homepage video background image", "wikirate grey logo",
+    #  "partner logos"].each do |name|
+    #   ensure_card name, codename: codename(name),
+    #                     type_id: Card::ImageID,
+    #                     empty_ok: true
+    # end
```

Comment:
empty_ok was making things fail on staging, where the image cards already existed.  And it was failing without it on local.